src/Advection/Advection.jl

```diff
@@ -52,11 +54,11 @@ abstract type AbstractUpwindBiasedAdvectionScheme{B, FT} <: AbstractAdvectionSch
 # are constructed via metaprogramming. (The `advection_buffer` is the width of
 # the halo region required for an advection scheme on a non-immersed-boundary grid.)
 # An upper limit of `advection_buffer = 6` means we can build advection schemes up to
-# `Centered(order=12`) and `UpwindBiased(order=11)`. The list can be extended in order to
+# `Centered(order=10`) and `UpwindBiased(order=9)`. The list can be extended in order to
 # compile schemes with higher orders; for example `advection_buffers = [1, 2, 3, 4, 5, 6, 8]`
 # will compile schemes for `advection_buffer=8` and thus `Centered(order=16)` and `UpwindBiased(order=15)`.
 # Note that it is not possible to compile schemes for `advection_buffer = 41` or higher.
-const advection_buffers = [1, 2, 3, 4, 5, 6]
+const advection_buffers = [1, 2, 3, 4, 5]
```

Comment:
why?

Reply:
we should compile _past_ the recommended order. If order=9 is recommended we need order=11 to verify this.

Reply:
Okay, I guess I'll take this suggestion for https://github.com/CliMA/Oceananigans.jl/pull/4434, which, apart from this change, should be ready.